x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/use_create_analytics_form.ts

```diff
@@ -108,21 +116,30 @@ export const useCreateAnalyticsForm = (): CreateAnalyticsFormProps => {
           }
         ),
       });
-      setIsJobCreated(true);
-      refresh();
-      return true;
+
+      if (
+        creationResp?.dataFrameAnalyticsJobsCreated.length &&
```

Comment:
I don't think there is a risk of `creationResp` being `undefined` here, but it would be good to update the type of `ml.dataFrameAnalytics.createDataFrameAnalytics` so the response isn't `any`, it should be `PutDataFrameAnalyticsResponseSchema`

Reply:
The `?` can be removed from all of the `creationResp` uses

Reply:
Removed in 64fe11a2d3a9d571893ec587b2924bcff08c2b74